pkgs/development/libraries/libffi/default.nix

```diff
@@ -21,7 +16,13 @@ stdenv.mkDerivation rec {
     sha256 = "081nx7wpzds168jbr59m34n6s3lyiq6r8zggvqxvlslsc4hvf3sl";
   };
 
-  patches = [];
+  # Note: this package is used for bootstrapping fetchurl, and thus
```

Comment:
Better place but most other files have it at the top. Should be fine to change it though.